documentation/doc.yaml

```diff
@@ -920,6 +920,25 @@ annotations:
     - service
     version_min: "1.4"
     example: ['server-ssl: "true"']
+  - title: enable-grpc
+    type: bool
+    group: enable-grpc
+    dependencies: ""
+    default: "false"
+    description:
+      - Enables GRPC to pods.
```

Comment:
we also need here a note that this directly conflicts with  'server-ssl' annotation and that both of them can't be used at the same time